lib/msal-browser/src/interaction_client/RedirectClient.ts

```diff
@@ -83,6 +84,7 @@ export class RedirectClient extends StandardInteractionClient {
             let state: string;
             try {
                 state = this.validateAndExtractStateFromHash(responseHash, InteractionType.Redirect);
+                BrowserUtils.clearHash(window);
```

Comment:
Also regression?

Reply:
For this and line 25 above, neither of these fit with the visibility changes (private/protected) that you have in the rest of the PR.  Is this incorporating a change from a different commit/PR that you've brought in?

Reply:
These are merge conflicts from PRs that were merged after this branch was created, resolving them now.

Reply:
Also, the cache changes are meant to be done, as they are part of the "extensibility" changes